features/openshift3.adoc

```diff
@@ -0,0 +1,41 @@
+= OpenShift 3
+:page-layout: features
+:page-product_id: jbt_core
+:page-feature_id: openshift3
+:page-feature_image_url: images/openshift_icon_256px.png
+:page-feature_highlighted: true
+:page-feature_order: 2
+:page-feature_tagline: The next gen Cloud
+
+== OpenShift 3
+=== Discover the new OpenShift
+image::images/openshift3_create_app.png[]
+
+JBoss Tools presents to you a TechPreview version or the OpenShift 3 tooling for Eclipse.
```

Comment:
Original: JBoss Tools presents to you a TechPreview version or the OpenShift 3 tooling for Eclipse.
Recommended: JBoss Tools presents a Technical Preview version of the OpenShift 3 Tooling for Eclipse.